src/gamecontroller.cs

```diff
@@ -1922,4 +1996,50 @@ enum GCInputDirectional {
 		[Field ("GCInputDirectionalCardinalDpad")]
 		CardinalDpad,
 	}
+
+	delegate GCVirtualControllerElementConfiguration GCVirtualControllerElementUpdateBlock (GCVirtualControllerElementConfiguration configuration);
+
+	[NoTV, NoMac, NoWatch, iOS (15,0), MacCatalyst (15,0)]
+	[BaseType (typeof(NSObject))]
```

Comment:
Really minor, missing space:

```suggestion
	[BaseType (typeof (NSObject))]
```

Reply:
Good catch!